integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
IMO, this is reasonable, but it's really testing for "internet
availability" not "network access" (which is the real heart of what we want
to gate these "network" tests on, right?)